### PR TITLE
use json layout format, make soundbutton class

### PR DESCRIPTION
**Pull request by adam-j-turner**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -38,72 +96,79 @@ class Container extends React.Component {
...
 
+  startProgress() {
+    var _this = this
+    this.interval = setInterval(
+      function() {
+        _this.setState({
+          progress: _this.currentAudio.currentTime * 100 / _this.currentAudio.duration 
+        })
+      },
+      40
```
Comment (CollinEMac) [inline src/App.js]: What does this do?

File: src/App.js [hunk 2]
```diff
@@ -1,31 +1,89 @@
...
+    this.playing = false
     this.interval = null
 
-    this.changeEditMode = this.changeEditMode.bind(this)
-    this.playSound = this.playSound.bind(this)
+    this.handleEditModeChange = this.handleEditModeChange.bind(this)
+    this.handleSoundButtonClick = this.handleSoundButtonClick.bind(this)
+  }
+
+  importBulk(r) {
```
Comment (CollinEMac) [inline src/App.js]: Can you write some comments for this method? For example, what is `r`? What does it return?

File: src/App.js (hunk 1 above)
Comment (adam-j-turner) [reply src/App.js]: Updates the progress bar every 40 ms.

File: src/App.js (hunk 2 above)
Comment (CollinEMac) [reply src/App.js]: Bump.

Comment (adam-j-turner) [reply src/App.js]: Added a comment and made it more readable.

**Pull request merged**